src/components/Causes/CauseCard/CauseCard.js

```diff
@@ -36,12 +38,20 @@ const CauseCard = props => {
 }
 
 CauseCard.propTypes = {
-  causeName: PropTypes.string,
-  donatedAmount: PropTypes.number,
-  targetAmount: PropTypes.number,
-  numberOfDonors: PropTypes.number,
-  daysToGo: PropTypes.string,
-  organization: PropTypes.string
+  cause: PropTypes.shape({
```

Comment:
I believe the build test is failing because of the required PropTypes.  The GraphQL query is returning _all_ results as PropType.string so if we expect numbers it will fail to compile.  We either need to convert the query results to numbers or expect PropTypes.string.  This includes fields that were input as numbers such as donatedAmount.

Reply:
Thanks @azure-may let me resolve this.